pom.xml

```diff
@@ -3605,5 +3607,144 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>ci-test</id>
+            <build>
+                <plugins>
```

Comment:
Sorry about the failing build : it didn't fail in my cursory testing

We actually need to wrap this in a `<pluginManagement>` tag otherwise activating the profile enables the non-disabled plugin executions on projects that do not need them and the build fails 
Also we probably want to disable the remaining executions I'll add the necessary code in a comment at the right line